app/views/categories/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="flex flex-col items-center my-6 mx-4">
+  <h1 class="text-3xl font-medium mb-6"><%= @category.name %></h1>
+  <% if @total == 0 %>
+  <p class="text-gray-600">No expenses found</p>
+  <img src="https://cdn-icons-png.flaticon.com/512/8541/8541407.png" alt="No expenses found" class="w-32 h-32 my-10">
+<% else %>
+  <p class="text-xl font-medium mb-6"><%= @total %></p>
+
+  <div class="grid grid-cols-1 gap-6">
+    <% @expenses.each do |expense| %>
+      <div class="flex flex-col items-center p-6 rounded-lg bg-white shadow-lg">
+        <p class="text-lg font-medium mb-3"><%= expense.name %></p>
+        <p class="text-lg font-medium mb-3"><%= expense.amount %></p>
+        <p class="text-lg font-medium mb-3"><%= expense.created_at.strftime("%d %b %Y") %></p>
+      </div>
+    <% end %>
+  </div>
+<% end %>
+  <%= link_to "Add new expense", new_category_expense_path(@category), class: "bg-blue-400 text-white py-2 px-4 rounded-lg m-6" %>
+  <%= link_to "Back", categories_path, class: "text-blue-500 py-2 px-4 rounded-lg m-6" %>
+</div>
```

Comment:
- [ ] As previously pointed out, please ensure your design matches the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/219108466-059ae275-d0e3-4d18-8bc4-d75b1e153d54.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/219107076-18d94388-1256-4c3a-82c7-98208c53601c.png)

Reply:
This was handled 😉 
![Screenshot 2023-02-15 at 13 43 18](https://user-images.githubusercontent.com/45739963/219122942-62409ab3-061e-4d97-b329-7ecb7d7dca3b.png)